templates/configurations/timeRange.interface.hbs

```diff
@@ -41,5 +47,12 @@
 <div v-else class="form-group">
   <label>Default value</label>
   <div class="text-danger" v-if="errors.requiredField"> \{{ errors.requiredField }} </div>
-  <input type="time" class="form-control basic-time-picker" v-model.trim="value" />
+  <div class="fl-time-range form-html">
```

Comment:
This doesn't look right to me. https://github.com/Fliplet/fliplet-api/pull/5725/files#diff-4612bda789533f214313c060a4fa54e08258786d64a0f1355e6bcb7a64d96129R13-R29 says a different markup is needed. Can you also share a screenshot of this one it's correctly done?

Reply:
![image](https://github.com/Fliplet/fliplet-api/assets/128052346/30c951db-2d8b-43fa-918e-5ee7a9653b67)

this is for the custom option

Reply:
@armine-fliplet Then let's use the same UI markup you've defined for `Fliplet.UI.TimeRange`. It'll ensure consistent look and feel with the time range field as well.